### ISSUE TITLE
Missing library libssl.so.1.1

### ISSUE THREAD

**Issue by numberformat** [itzg/docker-minecraft-bedrock-server#404]
Starting with this command

root@minecraft:~/minecraft# docker run -dit --name='minecraft2' -p 19132:19132/udp -e EULA=TRUE -e VERSION=1.16 --restart=unless-stopped itzg/minecraft-bedrock-server

./bedrock_server-1.16.20.03: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
2024-03-21T02:00:19.163Z        WARN    mc-server-runner        Minecraft server failed. Inspect logs above for errors that indicate cause. DO NOT report this line as an error.   {"exitCode": 127}
2024-03-21T02:00:19.163Z        INFO    mc-server-runner        Done

**Comment by itzg** [comment]
I am unable to recreate this issue. I'm hoping somebody else will have some advice for you.

**Comment by karlchu** [comment]
Just some information that may be helpful: I get the same error when I run the container under the x86_64 architecture. However, when I run the container under the arm64 architecture, it works fine.

**Comment by numberformat** [comment]
> Just some information that may be helpful: I get the same error when I run the container under the x86_64 architecture. However, when I run the container under the arm64 architecture, it works fine.

Yes. I am running it on x86_64

**Comment by weagle08** [comment]
what OS?

**Comment by karlchu** [comment]
> what OS?

ubuntu:22.04

**Comment by weagle08** [comment]
you should be able to simply run `sudo apt install openssl` to get the required shared lib

**Comment by karlchu** [comment]
Yeah... no. The problem is newer Ubuntu comes with a newer version of libssl (3.x?), and it breaks something. If anything, one needs to downgrade to 1.1, and it is not that simple.
